CHANGELOG.rst

```diff
@@ -4,7 +4,19 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
-1.0.0-beta.5-SNAPSHOT (2021-11-22)
+1.0.0-beta.6 (2021-12-07)
```

Comment:
MAJOR NIT 💂:  
```suggestion
1.0.0-beta.6 (2021-12-08)
```

Reply:
ouh.. here I also forgot the snapshot tag..

Reply:
```suggestion
1.0.0-beta.6-SNAPSHOT (2021-12-08)
```